### PR TITLE
BaSP-TG-93: Employee CREATE, GET and DELETE functions with shared components

### PR DESCRIPTION
**Pull request by FefeTroanes**
This is our request to merge this branch with the master one.
Here we updated the functions related to the employees developed in the past week with shared components, such as the table, buttons, sidebar, and so on.

### CODE REVIEW

File: src/Components/Employees/EditEmployee/editEmployee.jsx [hunk 1]
```diff
@@ -1,14 +1,32 @@
...
+  };
+  const valueActiveChange = (e) => {
+    return setActive(e.target.value);
+  };
   useEffect(() => {
-    fetch(`${process.env.REACT_APP_API_URL}/employees/${employeeId}`)
+    fetch(`${process.env.REACT_APP_API_URL}/employees/${props.id}`)
       .then((response) => response.json())
       .then((response) => {
+        console.log(response);
```
Comment (nicolascostanza) [inline src/Components/Employees/EditEmployee/editEmployee.jsx:29]: borra el console.log

File: src/Components/Employees/EmployeeForm/addEmployee.jsx [hunk 2]
```diff
@@ -11,7 +13,9 @@ const AddEmployee = () => {
   const [dob, setDob] = useState('');
   const [password, setPassword] = useState('');
   const [phone, setPhone] = useState('');
-  const [active, setActive] = useState(false);
+  const [active, setActive] = useState();
```
Comment (nicolascostanza) [inline src/Components/Employees/EmployeeForm/addEmployee.jsx:16]: Este no deberia arrancar en false ? es pregunta

File: src/Components/Employees/EditEmployee/editEmployee.jsx (hunk 1 above)
Comment (nicolascostanza) [inline src/Components/Employees/EditEmployee/editEmployee.jsx:29]: este console.log borralo

File: src/Components/Employees/EmployeeForm/addEmployee.jsx [hunk 3]
```diff
@@ -11,7 +13,9 @@ const AddEmployee = () => {
   const [dob, setDob] = useState('');
   const [password, setPassword] = useState('');
   const [phone, setPhone] = useState('');
-  const [active, setActive] = useState(false);
```
Comment (nicolascostanza) [inline src/Components/Employees/EmployeeForm/addEmployee.jsx:14]: lo mismo del state aca, es consulta

File: src/Components/Employees/EmployeeForm/addEmployee.jsx [hunk 4]
```diff
@@ -130,16 +149,21 @@ const AddEmployee = () => {
...
+          <Dropdown
+            title="Active"
+            value={active}
+            onChange={valueActiveChange}
+            placeholder={'Select status'}
+            width={'170px'}
+          >
+            <option value="true">True</option>
+            <option value="false">False</option>
+          </Dropdown>
```
[the remark this answers is not in the document]
Comment (FefeTroanes) [reply src/Components/Employees/EmployeeForm/addEmployee.jsx:161]: Dale, lo tenemos en cuenta

File: src/Components/Employees/EmployeeForm/addEmployee.jsx (hunk 2 above)
Comment (FefeTroanes) [reply src/Components/Employees/EmployeeForm/addEmployee.jsx:16]: Si le pongo `false` el dropdown se queda sin placeholder

**Comment by Clarosluciano** [comment]
# Testing results

Tested locally on: https://alfonso-trackgenix-diwlytgfs-basp-m2022.vercel.app/employees
Validations: pass

# Bugs found
When the edit process fails, the data is missing
image
**Expected behavior:** the data isn't miss for that the user can edit only the correspondent data

**Comment by FefeTroanes** [comment]
> # Testing results
> 
> Tested locally on: https://alfonso-trackgenix-diwlytgfs-basp-m2022.vercel.app/employees Validations: pass
> # Bugs found
> 
> When the edit process fails, the data is missing image **Expected behavior:** the data isn't miss for that the user can edit only the correspondent data

Fixed! 🙌

**Review by Clarosluciano — APPROVED** [review]
Functionalities: pass
Validations: pass

**Review by martinalbrnz — APPROVED** [review]
Good job!

**Review by ilcosme — APPROVED** [review]
Good to go! Rush B

**Review by martinalbrnz — APPROVED** [review]
Niceee

**Pull request merged**